src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -4432,6 +4432,24 @@ def generate_params_file(cmd, file, no_restore=None, outdir=None, outfile=None,
         logger.error("az bicep generate-params could not be executed with the current version of Bicep CLI. Please upgrade Bicep CLI to v%s or later.", minimum_supported_version)
 
 
+def lint_bicep_file(cmd, file, no_restore=None, diagnostics_format=None):
+    ensure_bicep_installation(cmd.cli_ctx)
+
+    minimum_supported_version = "0.7.4"
```

Comment:
I have seen many duplicate definitions `minimum_supported_version = "0.7.4"` at present. Could we abstract them into a uniformly configurable constant? This is convenient for future maintenance.

Reply:
Although I agree with your comment, I don't think this PR should have changes for other places in code. I'm going to create a follow up PR to address this request, if that's OK.

Can I ask to finish review with my comment in mind, please?

Reply:
OK, looks good to me